pyanaconda/ui/gui/spokes/root_password.py

```diff
@@ -118,9 +119,11 @@ def initialize(self):
         self.checker.add_check(self._validity_check)
         self.checker.add_check(self._ascii_check)
 
-        # the password entries should be sensitive on first entry to the spoke
-        self.password_entry.set_sensitive(True)
-        self.password_confirmation_entry.set_sensitive(True)
+        # the password entries should be sensitive only if root is not locked from kickstart
+        allow_changes = self._users_module.CanChangeRootPassword or self.checker.policy.changesok
```

Comment:
I think this should be "and" instead of "or" - eq. to change the root password, both the module and the policy need to allow that. Or was it done differently in the past ?

Reply:
tl;dr OR is right here.

The names are kind of misleading. Either can allow entering the spoke. `CanChangeRootPassword` is actually kickstart's `seen`, just inverted, so that one means "yes you can change it, changes policy does not apply".